pcawg_metadata_parser/generate_release.py

```diff
@@ -316,16 +401,18 @@ def generate_variant_calling_info(pilot_tsv, variant_calling, vcf):
         pilot_tsv[get_formal_vcf_name(v)+'_repo'] = []
         pilot_tsv[get_formal_vcf_name(v)+'_gnos_id'] = []
         pilot_tsv[get_formal_vcf_name(v)+'_file_name_prefix'] = []
+        pilot_tsv['is_aug2015_'+get_formal_vcf_name(v)] = []
         for specimen in variant_calling:
             if specimen.get(get_formal_vcf_name(v)):
                 pilot_tsv[get_formal_vcf_name(v)+'_repo'].append(specimen.get(get_formal_vcf_name(v)).get('gnos_repo'))
```

Comment:
This is reported by Christina, for multiple tumor specimens, all of the calls are in one singe GNOS, so *gnos_repo* should be a string (not a list of strings), it is the same as *gnos_id*